arangod/RestHandler/RestMetricsHandler.cpp

```diff
@@ -152,7 +149,17 @@ RestStatus RestMetricsHandler::execute() {
     return RestStatus::DONE;
   } else {
     std::string result;
-    metrics.toPrometheus(result);
+    auto mode = metrics::CollectMode::Local;
+    if (it = values.find("mode"); it != values.end()) {
+      if (it->second == "trigger_global") {
+        mode = metrics::CollectMode::TriggerGlobal;
+      } else if (it->second == "read_global") {
+        mode = metrics::CollectMode::ReadGlobal;
+      } else if (it->second == "write_global") {
+        mode = metrics::CollectMode::WriteGlobal;
+      }
```

Comment:
maybe use `frozen::map<std::string_view, metrics::CollectMode, 4>`?
I'd also consider moving this into `CollectMode.h`

Reply:
Hmm, ok

Reply:
Done